packages/react/ds/src/link/link.tsx

```diff
@@ -15,7 +14,6 @@ export function Link({
   noUnderline,
   noVisited,
   external,
-  dark = false,
```

Comment:
if we don't use the `dark` property anymore I think we also need to `link.schema.ts`